presto-cache/pom.xml

```diff
@@ -133,4 +133,24 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
```

Comment:
please ignore change in this file, just to make the build work with Alluxio client on snapshot versions.
once release version is cut, will revert the change